css/properties/-webkit-mask-attachment.json

```diff
@@ -0,0 +1,54 @@
+{
+  "css": {
+    "properties": {
+      "-webkit-mask-attachment": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/-webkit-mask-attachment",
+          "support": {
+            "webview_android": {
+              "version_added": "2.1"
+            },
+            "chrome": {
+              "version_added": "24"
```

Comment:
The table says it was supported in Chrome (since an unknown version), then removed in Chrome 24.

Reply:
Ah, OK. I misunderstood what those entries meant. Fixed!